hawkrest/__init__.py

```diff
@@ -61,7 +86,7 @@ def authenticate(self, request):
 
         try:
             receiver = Receiver(
-                lookup_function,
+                lookup_credentials,
```

Comment:
Shouldn't this pass in the user-overiddable `credentials_lookup` function instead?